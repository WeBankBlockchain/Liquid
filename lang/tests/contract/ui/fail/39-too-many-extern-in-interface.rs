use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        fn bar(&self);
    }

    extern "liquid" {
        fn baz(&self);
    }
}

fn main() {}
