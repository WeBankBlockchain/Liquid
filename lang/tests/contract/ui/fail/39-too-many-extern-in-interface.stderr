error: interface module must have exactly one `extern` block
  --> $DIR/39-too-many-extern-in-interface.rs:9:5
   |
9  | /     extern "liquid" {
10 | |         fn baz(&self);
11 | |     }
   | |_____^
